data/json/furniture_and_terrain/terrain-flora.json

```
@@ -480,6 +480,16 @@
       "items": [ { "item": "stick_long", "count": [ 3, 10 ] }, { "item": "splinter", "count": [ 15, 30 ] } ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_tree_dead_warped",
+    "copy-from": "t_tree_dead",
+    "looks_like": "t_tree_dead",
+    "symbol": "G",
+    "color": "brown",
+    "name": "warped dead tree",
+    "description": "This deciduous tree's branches are warped into a spiral.  It is pretty clear that it died due to whatever caused this."
```

```suggestion
    "description": "This deciduous tree's branches are warped into a spiral.  This appears to have killed it."
```
Or something like this, the other formulation feels a bit too convoluted